UIMentoringSession/MenteesChallenges/Pets Main Card UI Chellenge/OleksandrKharchenko/AnimalsCells/Olkha_PetsModels.swift

```diff
@@ -0,0 +1,94 @@
+import Foundation
+import UIKit
+
+struct OlKha_PetsViewModel {
+    let name: String
+    let age: String
+    let breadAnimal: String
+    let type: AnimalsTypes
+    let animal: UIImage
+}
+
+struct OlKha_FilterViewModel {
+    let title: String
+    let type: AnimalsTypes
+    let animal: UIImage
+}
+
+let itemsFilter: [OlKha_FilterViewModel] = [
+    .init(title: "All",
+          type: .allPets,
```

Comment:
Always try to load 1 or 2 images for the test, it is redundant to load all test images of all cats and dogs. Just 1 or a maximum 2 cats is enough